staging/src/k8s.io/apiserver/pkg/storage/util.go

```diff
@@ -72,8 +72,8 @@ func NoNamespaceKeyFunc(prefix string, obj runtime.Object) (string, error) {
 	return prefix + "/" + name, nil
 }
 
-// hasPathPrefix returns true if the string matches pathPrefix exactly, or if is prefixed with pathPrefix at a path segment boundary
-func hasPathPrefix(s, pathPrefix string) bool {
+// HasPathPrefix returns true if the string matches pathPrefix exactly, or if is prefixed with pathPrefix at a path segment boundary
+func HasPathPrefix(s, pathPrefix string) bool {
```

Comment:
Is this used anywhere else? Can it be moved into the new package instead of being made public?

Reply:
yes, it's only used in cacher. And has moved it to cacher package instead of being made public. :)